servers/quarkus-server/src/main/java/org/projectnessie/server/config/DynamoVersionStoreConfig.java

```diff
@@ -15,12 +15,13 @@
  */
 package org.projectnessie.server.config;
 
-import io.quarkus.arc.config.ConfigProperties;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
 import org.projectnessie.versioned.dynamodb.DynamoStoreConfig;
 
```

Comment:
Think that'll cause a spotless-check error

Reply:
I thought this would have been picked up https://github.com/projectnessie/nessie/blob/b784d4d0290536644c4ed27163520a4dbe8d0f0d/pom.xml#L1325-L1330 but that doesn't seem to be the case